src/PgSQLApplication.php

```diff
@@ -25,4 +28,19 @@ public function __construct(array $config, Logger $logger)
 
         parent::__construct($config, $logger, $configDefinition);
     }
+
+    public function reorderColumns(SplFileInfo $csv, array $items): array
```

Comment:
k čemu to slouží? nevidím tu použití nikde

Reply:
Volá se to někde v předcích, když to takhle nepřetížím, padne to na tom, že neexistuje manifest :-/